io.catenax.return_request/2.0.0/ReturnRequest.ttl

```diff
@@ -0,0 +1,245 @@
+
+:TimePeriodEntity a samm:Entity ;
+   samm:preferredName "time period"@en ;
+   samm:description "Entity defining a range of dates to define a time period."@en ;
+   samm:properties ( :startDate :endDate ) .
+
+:IdentificationEntity a samm:Entity ;
+   samm:preferredName "identification entity"@en ;
+   samm:description "Entity encapsulating different unique identifier of the product (manufacturingPartID, OeNumber and/or serialNumber)"@en ;
+   samm:properties ( :manufacturingPartID :oeNumber :serialNumber ) .
+
+:missingParts a samm:Property ;
+   samm:preferredName "missing parts"@en ;
+   samm:description "Completeness of the product, e.g. missing parts are not acceptable."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:dismantled a samm:Property ;
+   samm:preferredName "dismantled"@en ;
+   samm:description "Information on whether the product has been dismantled."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:mechanicalDamage a samm:Property ;
+   samm:preferredName "mechanical damage"@en ;
+   samm:description "Information on mechanical damage to the part, e.g. two broken ribs or part deformation indicate mechanical damage."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:corroded a samm:Property ;
+   samm:preferredName "corroded"@en ;
+   samm:description "Information on whether the product has corrosion."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:discolored a samm:Property ;
+   samm:preferredName "discolored"@en ;
+   samm:description "Information on whether the product has been discoloured."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:mileage a samm:Property ;
+   samm:preferredName "mileage"@en ;
+   samm:description "Current mileage of the component"@en ;
+   samm:characteristic :NumericValueofMileage ;
+   samm:exampleValue 89000 .
+
+:startDate a samm:Property ;
+   samm:preferredName "start date"@en ;
+   samm:description "The first day of the period"@en ;
+   samm:characteristic :DateTimestamp ;
+   samm:exampleValue "2019-01-01"^^xsd:date .
+
+:endDate a samm:Property ;
+   samm:preferredName "end date"@en ;
+   samm:description "The last day of the period."@en ;
+   samm:characteristic :DateTimestamp ;
+   samm:exampleValue "2030-09-30"^^xsd:date .
+
+:manufacturingPartID a samm:Property ;
+   samm:preferredName "manufacturing part id"@en ;
+   samm:description "The unique identifier of the product from the manufacturer, characterizes the type of component"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "4S7R7002DB" .
+
+:oeNumber a samm:Property ;
+   samm:preferredName "oe number"@en ;
+   samm:description "The unique identifier of the product from the original manufacturer, characterizes the type of component"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "8O8RKJZ7ER" .
+
+:serialNumber a samm:Property ;
```

Comment:
use of serialPart (localIdentifiers)

Reply:
Is a simple change of naming sufficient? (serialNumber->localIdentifiers)

Reply:
I mean the use of the aspect serial part. Best to open the serial part model seperatly to see the redundant fields which can be linked.

Reply:
If the aspect serial part is used, then more shared aspects are linked in the model (shared.part_site_information_as_built, business_partner_number, etc.). This means that the prefix in these shared aspect involved also needs to be updated to eclipse 2.1.0 version. I will submit multiple PRs to apply for these changes. Is it ok?

Reply:
These will be updated anyway to 2.1.0 due to recent changes (#557 #581 #571). You can include the current version already, but be aware to change the prefix to the higher version before the PR is merged.